### DISCUSSION TITLE
There is literally no reason that Magnetized and Relax are incompatible.

### DISCUSSION THREAD

**Discussion by Paladynee** [ppy/osu#17782] [Q&A] [answered]
Magnetized (aka. Aim Assist) used to be compatible with Relax and i used it a lot. But now it isn't, I don't see any reason why would relax which only hits objects automatically would be incompatible with a mod that modifies where objects are.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/discussions/17681

**Reply by Paladynee** [reply]
It being not exactly autoplay should be more than enough for the devs to allow this. These mods arent even ranked. Just let us do it, no point not letting us. Even if it's auto, who cares?

**Reply by smoogipoo** [reply]
> These mods arent even ranked

Every mod is ranked except autoplay. The concept of "ranked" mods doesn't really exist anymore.

The problem is that these could be used to abuse profile stats, such as total/ranked score and play count.
